### ISSUE TITLE
신입 모집에서의 기밀 유지 의무와 자치권 보장

### ISSUE THREAD

**Issue by maeng22** [maeng22/shangrila-constitution#3]
과거 신입 모집 과정에서 리크루팅과 관련 없는 회원들이 지원자의 개인정보를 보려고 하여 개인정보가 외부로 유출되는 일이 발생해, 이와 같은 일을 막고자 리크루팅 과정에서 개인정보의 기밀 유지 의무화와 이에 따른 임원진의 자치권 보장을 회칙에 명문화하고자 합니다.

현재 개정안에 따르면, 임원진의 합의 하에 신입 선발 과정에 도움을 주는 면접 도우미를 선정할 수 있으며, 면접 도우미와 임원진만이 리크루팅 지원자의 개인정보에 접근이 가능하고, 신입 모집 종료까지 관련 정보의 기밀 유지를 지키도록 되어 있습니다.

위 사안에 대해서 샹그릴라 회원 여러분들의 의견을 구합니다.

**Comment by maeng22** [comment]
#4

**Comment by eddie878** [comment]
임원진은 동아리 원의 동의를 받아 새로운 신입을 선발하는데 있어서 충분히 그 권한이 있지만 임원진끼리 합의하에 도우미를 선발하는 것은 투명성에 의문이 생길 수 있을 것 같습니다. 최소한 도우미를 임원진 내부에서 합의하여 뽑는 것이 아닌 모집 공지나 아니면 선발후 누가 도우미가 되었다는 것에 대해 공지는 있어야 한다고 생각합니다.

**Comment by codename-whiteroom** [comment]
"면접 도우미"의 투명성에 대한 의문 확인했습니다.
현재 구상하던 생각으로는 면접 경험이 존재하는 전년도 임원진 중 한명이 들어가는 것이 어떤가 생각했는데, 해당 경우에도 투명성 의문이 유지될까요? 
만일 그렇다면 매 학기 초 면접 도우미 모집 및 선발 절차를 거처야 하는데, 쉽지 않을 것 같아 위와 같이 제안해봅니다.

**Comment by eddie878** [comment]
누가 되었는지에 대한 공지가 되었을 때 그 이유도 간단히 설명이 된다면 투명성이 확보가 될 것 같습니다. 신입을 뽑는것은 충분히 동아리 차원에서 큰 권한이고 전임 임원진이었다는 이유만이 아니라 더욱 자세한 선발 이유가 추가된다면 좋을 것 같습니다.

**Comment by codename-whiteroom** [comment]
면접 도우미의 "선발 사유"라고 하는 것이 잘 와닿지 않습니다. 어떤 사유가 가능할지 간단한 예시가 있을까요?
"면접 경험을 통해 더 명확한 선발이 가능하다" 정도의 사유를 같이 제시할 수 있다고 생각하긴 합니다.

**Comment by eddie878** [comment]
"면접 경험을 통해 더 명확한 선발이 가능하다" 이것보다 더욱 알맞는 사유가 있어야 동아리원이 합의해서 뽑은 임원진 이외에도 면접에 들어갈 권한이 필요한 것 같습니다.  예를들어 "서브컬쳐 전반에 대한 박학한 지식이나 다른 사람을 분석하는데 뛰어남 등" 이런 이유가 있는게 더 좋지 않을까요?

**Comment by codename-whiteroom** [comment]
확실히, 면접 경험만을 가지고 도우미를 자처하기에는 약간의 어려움이 있을 수 있다고 생각합니다.
주어진 예시는 잘 확인했습니다. 해당 사유를 언급하면 자동적으로 면접 도우미에 대한 토론이 오갈 수 있겠군요

다만, 해당 경우 만일 적절하지 못한 면접 도우미를 선출했다 판단되는 경우에는 항의를 할 수 있는 시간이 있어야 하는데, 현실적으로 면접 진행이 1주차이기 때문에 항의 이후 새로운 면접 도우미를 선출하는 과정이 어렵다는 문제가 있을 수 있습니다. 방학중의 선출 이후 사람들이 처음 모이는 공식적 장소인 개강총회보다 면접이 빠르게 진행될 수도 있고요. 이에 대해서는 추가적인 논의가 필요해 보입니다.

우선 현재까지 나온 발언을 토대로
"면접 도우미 선발 자체는 자유롭게 진행하되, 도우미 선발 이후 선정 사유를 명확하게 제시하여야 한다."
라는 단서를 추가적으로 달아 두는 방향으로 회칙을 수정하고자 합니다. 괜찮을까요?

**Comment by eddie878** [comment]
네 괜찮을 것 같습니다. 이러한 사유들을 보면서 다음 리크루팅이 잘 이루어지기 위한 피드백을 받을 수 있는 한 부분으로써 작용할 수 있기에 제안 드렸습니다.
